Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -420,24 +420,38 @@ public struct LocalizationKey {
     public static let threeds2DAApprovalNegativeButton = LocalizationKey(key: "adyen.threeds2.DA.approval.negativeButton")
     /// Other options
     public static let threeds2DAApprovalActionSheetTitle = LocalizationKey(key: "adyen.threeds2.DA.approval.actionSheet.title")
-    /// Approve differently
+    /// Approve another way
```

Comment:
This wording feels off grammatically. Is this from the translators? If it's going to change from `approve differently`, should be like `approve via another way`

Reply:
Thanks for already looking in this PR :)
This is from the the copy writers. I'll share with you the design file by Arjen. We could always change this.